### PR TITLE
Sample PR - Issue2219 remove obsolete attributes

### PR DESCRIPTION
**Pull request by stevenaw**
A sample PR for Issue #2219 to visualize the impact of removing obsolete attributes.

### CODE REVIEW

**Comment by ChrisMaddock** [comment]
I'm in favour of merging this into 3.8. I appreciate it doesn't follow SemVer - but we don't follow SemVer to strictly anyway. We should just come to peace with that, and call what we do NUnitVer instead. 😄 

They've been obsolete since 3.0, and people have had plenty of time to refactor. What I _would_ like to see is everything that was obsoleted in 3.0 removed in one go, to minimise user disruption. As far as I can see, the only other thing is the `Is.StringContaining` variants - but it would be good to double check that.

File: src/NUnitFramework/tests/Attributes/PairwiseTests.cs [hunk 1]
```diff
@@ -38,13 +38,13 @@ public class LiveTest
             private PairCounter pairsTested = new PairCounter();
 
             [OneTimeSetUp]
-            public void TestFixtureSetUp()
+            public void OneTimeSetUp()
```
Comment (CharliePoole) [inline src/NUnitFramework/tests/Attributes/PairwiseTests.cs]: Unneeded but OK. We often refer to the OneTimeSetUp of a fixture this way in the docs.

File: src/NUnitFramework/tests/Attributes/SetUpFixtureAttributeTests.cs [hunk 2]
```diff
@@ -31,38 +31,22 @@ namespace NUnit.Framework.Attributes
 {
     public class SetUpFixtureAttributeTests
     {
-        [TestCase(typeof(Class1))]
-        [TestCase(typeof(Class2))]
-        [TestCase(typeof(Class3))]
-        [TestCase(typeof(Class4))]
+        [TestCase(typeof(TestSetupClass))]
+        [TestCase(typeof(TestSearDownClass))]
```
Comment (CharliePoole) [inline src/NUnitFramework/tests/Attributes/SetUpFixtureAttributeTests.cs]: Spelling.

**Comment by CharliePoole** [comment]
Looks good to me as well if you fix the misspelled class name.

**Comment by rprouse** [comment]
I am good with this too. As @CharliePoole says, the spelling needs fixing, but after that, I think we should merge. I like that this only changes things that are easy to update with a global search and replace. I would prefer to deal with changes that will require hand updating as a SemVer breaking change.

**Comment by rprouse** [comment]
@stevenaw if you want to resolve conflicts and fix that one spelling error, I think we can merge this and take the rest of the obsolete code in steps.

File: src/NUnitFramework/tests/Attributes/SetUpFixtureAttributeTests.cs (hunk 2 above)
Comment (stevenaw) [inline src/NUnitFramework/tests/Attributes/SetUpFixtureAttributeTests.cs]: Thanks, good catch

**Comment by stevenaw** [comment]
Thanks @rprouse @CharliePoole @ChrisMaddock .
Spelling corrected and conflicts resolved.

Build completes locally, but running the tests locally has 2 failures due to `SocketException`s. I'm assuming it's just related to this https://github.com/nunit/nunit/issues/2332#issuecomment-318535845

**Comment by ChrisMaddock** [comment]
Looks good - re-running CI. 🙂

**Comment by rprouse** [comment]
CI passed, merging

**Pull request merged**